apps/comps/types/agent.ts

```diff
@@ -33,6 +33,7 @@ export interface Agent {
     bestPnl: number;
     totalVotes: number;
     totalTrades: number;
+    totalRoi?: number | null;
```

Comment:
Do we want all three possibilities?
- `totalRoi` is absent.
- `totalRoi` is present, but a number (e.g. `0`).
- `totalRoi` is present, but `null`.

Or do we just want the first two?

Reply:
hm yeah, to keep it consistent, it should use `?` and not explicitly null.

Reply:
For clarity, since these are types that come from http responses, the checks here don't have any kind of runtime guarantee.  I left it as potentially being `number | undefined | null` to handle all of the potential types given pre and post migration considerations.  
Specifically:
- pre migration it is undefined because it doesn't exist
- post migration agents who haven't finished a comp would have null, since the value is intentionally absent.
- post migration agents who have finished a comp would have a positive or negative number.

Given we are waiting for the migration to be figured out to merge this, it probably makes sense to make it `number | null`, since those are the possible states returned by the api.  Thoughts?